drivers/hmis/spec/requests/hmis/ce/ce_opportunity_spec.rb

```diff
@@ -41,6 +41,7 @@
                 clientId
               }
             }
+            candidatesGeneratedAt
```

Comment:
`referral` starts resolving wrong (and tests in this file start failing) if `candidatesGeneratedAt` is moved down below `referral`. Relates to https://github.com/greenriver/hmis-warehouse/pull/5302